atm.js

```diff
@@ -1,21 +1,87 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+    $(".checkDepBtn").on("click", function(){
+      var deposit = parseInt($(".checkEntry").val());
+      var balance = $("#checkDisplay").text()
+      balance = parseInt(balance.replace("$", ""))
+      var checkingBalance = (deposit + balance)
+      $("#checkDisplay").text("$" + checkingBalance)
+    })
 
-}
+    $(".checkWdrwBtn").on("click", function(){
+      var withdrawal = parseInt($(".checkEntry").val())
+      var balance = $("#checkDisplay").text()
+      balance = parseInt(balance.replace("$", ""))
+      var checkingBalance = (balance - withdrawal)
+        if (checkingBalance >= 0) {
+          if (checkingBalance !== 0) {
+            $("#checkDisplay").text("$" + checkingBalance)
+            }
+          else {
+            $("#checkDisplay").text("$" + checkingBalance)
+            $("#checkDisplay").css("background-color", "#F52F4F");
+          }
+        }
+      })
 
-function checkingWithdrawal(amount){
+      $(".saveDepBtn").on("click", function(){
+        var deposit = parseInt($(".saveEntry").val());
+        var balance = $("#saveDisplay").text()
+        balance = parseInt(balance.replace("$", ""))
+        var savingsBalance = (deposit + balance)
+        $("#saveDisplay").text("$" + savingsBalance)
+      })
 
-}
+      $(".saveWdrwBtn").on("click", function(){
+        var withdrawal = parseInt($(".saveEntry").val())
+        var balance = $("#saveDisplay").text()
+        balance = parseInt(balance.replace("$", ""))
+        var savingsBalance = (balance - withdrawal)
+          if (savingsBalance >= 0) {
+            if (savingsBalance !== 0) {
+              $("#saveDisplay").text("$" + savingsBalance)
+              }
```

Comment:
This code looks very similar to the code above for the checking withdrawal. Can you think of a way to share the functionality between the two accounts?